dlt-connector/tsconfig.json

```diff
@@ -48,7 +48,13 @@
     "baseUrl": ".",                                 /* Base directory to resolve non-absolute module names. */
     "paths": {                                      /* A series of entries which re-map imports to lookup locations relative to the 'baseUrl'. */
       "@/*": ["src/*"],
+      "@arg/*": ["src/graphql/arg/*"],
+      "@enum/*": ["src/graphql/enum/*"],
+      "@input/*": ["src/graphql/input/*"],
+      "@resolver/*": ["src/graphql/resolver/*"],
+      "@scalar/*": ["src/graphql/scalar/*"],
       "@test/*": ["test/*"],
+      "@proto/*" : ["src/proto/*"],
```

Comment:
why do you store the model for the graphql-api outside the graphql-directory?

Reply:
because this model represent the gradido transaction on the tangle, so it don't really belong to the graphql-api.